### PR TITLE
fix(Forms): UI aligments of the ChildrenWithAge block

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/blocks/ChildrenWithAge/demos.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import * as Examples from './Examples'
 
 ### In Wizard
 
-All features are enabled in this example.
+All features, and additional questions(`enableAdditionalQuestions`) and custom translations, are enabled in this example.
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/blocks/ChildrenWithAge/demos.mdx:11]:
```suggestion
All features, and additional questions (`enableAdditionalQuestions`) and custom translations, are enabled in this example.
```

😉

**Review by tujoworker — APPROVED** [review]
Great work ❤️

And that's the new "default", right?

Screenshot 2024-10-15 at 12 48 14

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/blocks/ChildrenWithAge/demos.mdx (hunk 1 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/blocks/ChildrenWithAge/demos.mdx:11]: Haha, I'll never learn 😂

**Review by joakbjerk — APPROVED** [review]
Looks good, and much cleaner 🙏

**Comment by langz** [comment]
> Great work ❤️
> 
> And that's the new "default", right?
> 
> Screenshot 2024-10-15 at 12 48 14

Yeah, they don't seem to want any "assistance", they want the user to have to fill in all values(like everything is an active choice).

File: packages/dnb-eufemia/src/extensions/forms/blocks/ChildrenWithAge/ChildrenWithAgeTranslations.ts [hunk 2]
```diff
@@ -50,24 +57,30 @@ const enGB = {
     suffix: 'years old',
   },
   hasJointResponsibility: {
-    fieldLabel:
-      'Do you have joint parental responsibility for any of the children?',
+    fieldLabel: 'Do you pay child support?',
+    required: 'You must state whether you pay chid support.',
```
[the remark this answers is not in the document]
Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/blocks/ChildrenWithAge/ChildrenWithAgeTranslations.ts:61]: Great, thanks 💯

**Comment by joakbjerk** [comment]
Just noticed that the bottom part of the border of the last input disappears on hover, but I don't think it's this PR that is causing this 
Screenshot 2024-10-15 at 13 20 41

**Comment by langz** [comment]
> Just noticed that on hover, the bottom part of the border of the last input disappears on hover, but I don't think it's this PR that is causing this Screenshot 2024-10-15 at 13 20 41

Yeah, I've seen it myself, it's not just here, but in several other examples that we have.

File: packages/dnb-eufemia/src/extensions/forms/blocks/ChildrenWithAge/ChildrenWithAgeTranslations.ts [hunk 3]
```diff
@@ -50,24 +57,30 @@ const enGB = {
...
     fieldLabel: 'Joint parental responsibility',
   },
   usesDaycare: {
-    fieldLabel: 'Do you have SFO/AKS for any of the children?',
-    required:
-      'You must state whether you have SFO/AKS for any of the children.',
+    fieldLabel: 'Do you have expenses for SFO/AKS?',
+    required: 'You must state whether you have any expenses for SFO/AKS.',
+    helpText:
+      'State the total amount per month that you pay to Skolefritidsordningen (SFO) or Aktivitetsskolen (AKS). \n Kindergarten expenses are not to be included here.',
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/blocks/ChildrenWithAge/ChildrenWithAgeTranslations.ts:74]:
```suggestion
      'State the total amount per month that you pay to Skolefritidsordningen (SFO) or Aktivitetsskolen (AKS).\n\nKindergarten expenses are not to be included here.',
```

**Pull request merged**